go/tools/gazelle/gazelle/main.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	goPrefix = flag.String("go_prefix", "", "go_prefix of the target workspace")
 	repoRoot = flag.String("repo_root", "", "path to a directory which corresponds to go_prefix, otherwise gazelle searches for it.")
-	mode     = flag.String("mode", "print", "print, fix or diff")
+	mode     = flag.String("mode", "fix", "print, fix or diff")
```

Comment:
Fixed https://github.com/bazelbuild/rules_go/pull/128 to merge in all cases.

Added more info on each mode.